packages/components/src/progress-bar/index.tsx

```diff
@@ -43,3 +47,12 @@ function UnforwardedProgressBar(
 export const ProgressBar = forwardRef( UnforwardedProgressBar );
 
 export default ProgressBar;
+
+type Test1 = { asd: 123 };
+type Test2 = { kljk: 121233 };
+
+type Compute< T > = { [ K in keyof T ]: T[ K ] };
+
+type Test = Compute< Test1 & Test2 >;
+//   ^?
+declare const test: Test;
```

Comment:
Should we clean this up before shipping?

Reply:
Oops! I was demoing [twoslash](https://github.com/twoslashes/twoslash) and some TypeScript tricks for Lena, and forgot to clean up. My bad!